test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/StringTest.scala

```diff
@@ -441,4 +441,23 @@ class StringTest {
   @inline private def erased(array: Array[String]): Array[AnyRef] = {
     array.asInstanceOf[Array[AnyRef]]
   }
+
+  @Test def repeat(): Unit = {
+    assumeFalse("String#repeat requires JDK 11+",
+      executingInJVMOnJDK8OrLower)
```

Comment:
This will not be a sufficient guard, because the code doesn't compile on JDK 8 in the first place.

We have no precedent for implementing JDK stuff from JDK > 8. However, we had precedent in the past for JDK 7 and JDK 8, when we still supported building on JDK 6:
https://github.com/scala-js/scala-js/blob/617c4170fe9844eea0115b6ed59b2816201a70e7/project/Build.scala#L1369-L1382

Reply:
Attempted to configure build for JDK11 
https://github.com/scala-js/scala-js/pull/4137/commits/8bb6ff3f36b2c3f1ccebfc4a42a0465c10d6cc46